rules/prefer-includes.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const isIndexOfCallExpression = node => {
+	if (node.type !== 'CallExpression') {
+		return false;
+	}
+
+	const property = node.callee.property;
+
+	return property.name === 'indexOf';
```

Comment:
I'm not sure I totally understand. Could you provide an example in context where `indexOf(foo)` wouldn't be a `MemberExpression`?